blog/_posts/2017-09-07-third-dotty-milestone-release.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: blog
```

Comment:
Layout should be `blog-detail`, not `blog`.

Reply:
👍